src/main/java/org/spongepowered/api/entity/living/Living.java

```diff
@@ -47,6 +48,22 @@
  */
 public interface Living extends AttributeHolder, Entity, EntityProjectileSource, TeamMember {
 
+    /**
+     * {@link Keys#ABSORPTION}
+     * @return The amount of {@link org.spongepowered.api.effect.potion.PotionEffectTypes#ABSORPTION}
+     */
+    default BoundedValue.Mutable<Double> absorption() {
+        return this.requireValue(Keys.ABSORPTION).asMutable();
+    }
+
+    /**
+     * {@link Keys#ACTIVE_ITEM}
+     * @return The active item, such as food being eaten
+     */
+    default Value.Mutable<ItemStackSnapshot> activeItem() {
```

Comment:
@gabizou 

Can you give some guidance on how the Values in these interfaces should be returned to consumers?

Like, should they always be Value.Mutable? Just Value? Does it depend on what it is returning and what you can do with it in the game?

Reply:
I am preferring they're always `Mutable` from entities. It does make sense that they are reflective of their `Key` variables, but at the end of the day, the developer is likely to want to mutate the value and maybe offer it back, or present it to another entity after mutation.